### ISSUE TITLE
Project Brief

### ISSUE THREAD

**Issue by banukutlu** [psu-libraries/psulib_blacklight#15]
In GitLab by @ful105 on Feb 21, 2018, 15:32

**Comment by banukutlu** [comment]
In GitLab by @ful105 on Feb 21, 2018, 15:34

assigned to @rkt6

**Comment by banukutlu** [comment]
@cdmo will write a draft and send it to Binky and Ruth.

**Comment by cdmo** [comment]
@binkylush @ruthtillman, I started a [design document/project brief/one-pager](https://psu.app.box.com/notes/367191122132). The one question I had while writing it was: branding. Like, how are we branding?  Maybe just a "Catalog Search" heading like Temple? Obviously, not "The CAT" or "Search the BlackCat." But I do think this is one of those things that we should try to address now rather then later because naming things is charged.

Regardless, let me know what you think of the brief. I cribbed from the MVPs. I also created a new document to be a coverall for [requirements](https://staff.libraries.psu.edu/i-tech/daws/development/web-developmeent-requirements) (note I need to update browser stats!).

Note I included Course Reserves and My Account.
